tooling/reproducible/repro_common.sh

```diff
@@ -327,15 +327,24 @@ function processModuleInfo() {
   fi
 }
 
-# Remove windowns generate classes jdk/bin/server/classes.jsa & jdk/bin/server/classes_nocoops.jsa 
-function removeGeneratedClasses() {
+# Remove excluded files known to differ
+#  NOTICE - Vendor specfic notice text file
+#  cacerts - Vendors use different cacerts
+#  classlist - Used to generate CDS archives, can vary due to different build machine environment
+#  classes.jsa, classes_nocoops.jsa - CDS archive caches will differ due to Vendor string differences
+function removeExcludedFiles() {
   local JDK_DIR="$1"
-  local OS="$2"
-
-  if [[ "$OS" =~ CYGWIN* ]] || [[ "$OS" =~ Darwin* ]]; then
-    rm -rf "$JDK_DIR/bin/server/classes.jsa"
-    rm -rf "$JDK_DIR/bin/server/classes_nocoops.jsa"
-  fi
+  excluded="NOTICE cacerts classlist classes.jsa classes_nocoops.jsa"
```

Comment:
This is not quite right, the CDS archives will only differ on Windows and Mac, they "should be" identical on Linux.

Also, when comparing Temurin builds, the NOTICE and cacerts will be identical and should not be excluded.

I suspect you've added the CDS files for Linux due to the classlist ipv6 difference, that is a known issue, and we need to ensure we have the same ipv6 setting on the reproduce VM.

Reply:
Updated @andrew-m-leonard